deployment/streaming/scripts/define-streaming-augdiff-producer.sh

```diff
@@ -12,7 +12,7 @@ aws ecs register-task-definition \
     --network-mode awsvpc \
     --requires-compatibilities EC2 FARGATE \
     --cpu "1 vCPU" \
-    --memory "2 GB" \
+    --memory "${AUGDIFF_ECS_MEMORY} GB" \
```

Comment:
I'd make one of two changes focused on improving clarity:

1. Rename `AUGDIFF_ECS_MEMORY` to `AUGDIFF_ECS_MEMORY_GB`
2. Include the "multiplier" e.g. GB in the variable value, so that the user can specify in kb,mb,gb whatever works for them. This would be consistent with our use of other memory variables in this project, like `DRIVER_MEMORY`. 

Let's make the same change for both this var and `ECS_MEMORY`.